pom.xml

```diff
@@ -223,6 +361,53 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>xml-maven-plugin</artifactId>
+					<version>${xml-maven-plugin-version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.jboss.tattletale</groupId>
```

Comment:
What is this used for?

Reply:
Tattletale generates some very useful build-time reports on dependencies.  https://tattletale.jboss.org/ 
![image](https://user-images.githubusercontent.com/29655222/207107355-de5e7179-4646-4059-a197-12097e70a735.png)

Reply:
OK, can this be integrated on github somehow? I suppose I need to run it manually? What did you use it for?

Reply:
It would be good if we only had the bare minimum of dependencies that are strictly necessary for the code generation and FIXLatest respectively.

Reply:
@chrjohn - PR updated.